### ISSUE TITLE
Error: java.lang.IllegalArgumentException: Storage type 'plocal' is different by storage type in URL

### ISSUE THREAD

**Issue by xiaoqiangyi** [orientechnologies/orientdb#7923]
## Expected behavior  
a new db should be created. 

## Actual behavior  
en error is reported. 

## Steps to reproduce  

1) run OrientDB server on the local PC: bin/server.bat
2) run console: bin/console.bat
3) connect to remote server: orientdb> CONNECT remote:localhost root root 
4) create a new database and then comes the error:
orientdb {server=remote:localhost/}> CREATE DATABASE REMOTE:localhost/myremotetestdb root root PLOCAL  
Creating database [REMOTE:localhost/myremotetestdb] using the storage type  [plocal]... 
Error: java.lang.IllegalArgumentException: Storage type 'plocal' is  different by storage type in URL  orientdb {server=remote:localhost/}>

**Comment by luigidellaquila** [comment]
Hi @xiaoqiangyi 

It seems to be a matter of case, try with 

```
CREATE DATABASE remote:localhost/myremotetestdb root root plocal
```  

(with both "remote" and "plocal" in lowercase), it should work fine

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @xiaoqiangyi 

I just pushed a fix to also support REMOTE in uppercase. The fix will be released with 2.2.32

Thanks

Luigi
